modules/battery_manager/php/edit.class.inc

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * This file contains the NDB_Form_media class
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\battery_manager;
+
+/**
+ * Creates the Edit form for NDB_Form_battery_manager
+ *
+ * @category Loris
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Edit extends \NDB_Form
+{
+    public $skipTemplate = true;
```

Comment:
do we need this line if we are not using smarty templates?

Reply:
the line is explicitly telling it not to use smarty templates.

Reply:
but is it required if there are no smart templates in the module?

Reply:
i think its necessary because when i comment out this line, i get:
"The following errors occurred while attempting to display this page:
 Unable to load template 'file:menu_battery_manager.tpl'"